src/libraries/Microsoft.Extensions.Http/src/HttpClientFactoryOptions.cs

```diff
@@ -103,5 +109,58 @@ public TimeSpan HandlerLifetime
 
         internal bool SuppressDefaultLogging { get; set; }
         internal List<Action<HttpMessageHandlerBuilder>> LoggingBuilderActions { get; } = new List<Action<HttpMessageHandlerBuilder>>();
+
+        private List<Action<HttpMessageHandlerBuilder>>? _httpMessageHandlerBuilderActions;
+        private List<Action<IPrimaryHandlerBuilder>>? _primaryHandlerActions = new List<Action<IPrimaryHandlerBuilder>>();
+        private List<Action<IAdditionalHandlersBuilder>>? _additionalHandlersActions = new List<Action<IAdditionalHandlersBuilder>>();
+
+        internal IReadOnlyList<Action<IPrimaryHandlerBuilder>> PrimaryHandlerActions => _primaryHandlerActions!;
```

Comment:
Why the nullable suppression?  I'd have expected instead these would be nullable but then you'd annotate members like  MergedToHandlerBuilderActions with `[MemberNotNull(...)]` to indicate that they can be used as a guard.